lib/assets/stylesheets/modules/_search.scss

```diff
@@ -60,50 +61,67 @@ html.has-search-results-open {
     overflow-x: scroll;
     -webkit-overflow-scrolling: touch;
     -ms-overflow-style: none;
-    @include media(tablet) {
+      @include govuk-media-query(tablet) {
       padding: 22px 20px 0 30px;
       top: 0;
       // The width of the sidebar
       left: 330px;
     }
+
+    a:link, a:visited {
+      text-decoration: none;
+      color: $govuk-link-colour;
+    }
+
+    a:hover {
+      text-decoration: underline;
+
+      &:focus {
+        text-decoration: none;
+      }
+    }
+    a:focus {
+      outline: none;
+      @include govuk-focused-text;
+    }
   }
 .search-results__inner {
   position: relative;
-  background: $white;
-  border-top: 1px solid $grey-2;
+  background: govuk-colour("white");
+  border-top: 1px solid govuk-colour("mid-grey");
   max-width: 40rem;
   padding: 15px;
-  @include media(tablet) {
+    @include govuk-media-query(tablet) {
     padding: 15px 200px 50px 40px;
-    border: 1px solid $grey-2;
+    border: 1px solid govuk-colour("mid-grey");
     &::after {
       content: '';
       position: absolute;
       top: 10px;
       left: -9px;
       width: 10px;
       height: 20px;
-      background: no-repeat file-url('search-result-caret.svg') center right;
+      background: no-repeat url('/images/search-result-caret.svg') center right;
       background-size: contain;
     }
   }
 }
 .search-results__title {
-  @include bold-27;
-  margin-bottom: $gutter;
+  @include govuk-font($size: 27);
+  margin-bottom: govuk-spacing(6);
 }
 .search-results__close {
-  @include core-16;
+  @include govuk-font($size: 16);
   position: absolute;
   top: 18px;
   right: 20px;
   appearance: none;
   background: none;
   border: 0;
   padding: 0;
+
   &:focus {
-    outline: 3px solid $focus-colour;;
-    outline-offset: 0;
+    background-color: $govuk-focus-colour;
```

Comment:
The close link is also including the default user agent focus style

<img width="1485" alt="Screen Shot 2019-09-04 at 11 10 21" src="https://user-images.githubusercontent.com/121939/64246408-a5345580-cf04-11e9-9567-d120a8f90f8e.png">